### PR TITLE
Add Nuo to Finance group. Create Exchanges group and add entries

### LINKED ISSUE #1: Add 10 New Companies Or Projects To The Use Dai List
Add 10 new companies or projects which accept Dai as payments, donations, or enable its use in some productive way on their platform. Your own company or Freelance business can be included.

Please do not make assumptions about any companies for which it is not clear Dai can be accepted.

Please review the guidelines here before including a company: https://github.com/apbendi/use-dai/blob/master/CONTRIBUTING.md#guidelines

### PR DESCRIPTION
**Pull request by kolya182**
Partially close https://github.com/apbendi/use-dai/issues/1

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -51,19 +52,28 @@
...
 * [Compound](https://compound.finance/) - Earn interest on Dai via algorithmic Money Markets.
 * [Dharma](https://www.dharma.io) - Lend your Dai and earn interest.
 * [Maker](https://cdp.makerdao.com/) - Get loans in Dai against collateralized ETH.
-* [Uniswap](https://uniswap.io/) - Decentralized protocol for swapping ETH, Dai and other tokens.
+* [Nuo](https://app.nuo.network/) - Simplest way to margin trade.
+
+## Trading
 
+* [Uniswap](https://uniswap.io/) - Decentralized protocol for swapping ETH, Dai and other tokens.
+* [KyberSwap](https://kyberswap.com/) - A simple way to exchange tokens. No deposit, oo orderbook, competitive spreads.
```
Comment (apbendi) [inline README.md]: > No deposit, oo orderbook

Minor spelling mistake

File: README.md [hunk 2]
```diff
@@ -51,19 +52,28 @@
...
-## Finance, Investment, and Trading
+## Finance, Investment
 
 * [Compound](https://compound.finance/) - Earn interest on Dai via algorithmic Money Markets.
 * [Dharma](https://www.dharma.io) - Lend your Dai and earn interest.
 * [Maker](https://cdp.makerdao.com/) - Get loans in Dai against collateralized ETH.
-* [Uniswap](https://uniswap.io/) - Decentralized protocol for swapping ETH, Dai and other tokens.
+* [Nuo](https://app.nuo.network/) - Simplest way to margin trade.
+
+## Trading
```
Comment (apbendi) [inline README.md]: I don't hate the idea of breaking this out, but I'd like to adhere to alphabetical order, both for the ordering of sections and for the ordering of items within the section. Do you mind fixing that?

File: README.md [hunk 3]
```diff
@@ -51,19 +52,28 @@
...
+* [dYdX](https://trade.dydx.exchange/) - The most powerful open trading platform for crypto assets.
+* [Neutral](https://dapp.neutralproject.com/) - Neutral Dollar is a metastable basket of stablecoins. A digital dollar with lower volatility and globally shared liquidity.
 
 ## Freelancers
 
 ## Other
 
 * [Gitcoin](https://gitcoin.co/) - Platform for crowdfunding and freelance developers for your software projects.
+* [Living Room of Satoshi](https://www.livingroomofsatoshi.com/) - Pay with confidence at Australia's largest crypto payments gateway.
+* [CelerX](https://celerx.app/) - Play eSports games, win crypto prizes. Have fun. Play to earn!
```
Comment (apbendi) [inline README.md:86]: Nice additions! 👍  Again, only request would be to use alphabetical order.

**Review by apbendi — CHANGES_REQUESTED** [review]
Thanks so much for your contribution. This is great! Congrats on being the first person to open a pull request on this brand new project 🎉 

I've added some minor requests, but overall things look great. One more nitpick: do you mind squashing these down to one commit message, with the current commit messages as bulleted items below the fold? Thanks again!

File: README.md (hunk 1 above)
Comment (kolya182) [reply README.md]: Fixed here https://github.com/apbendi/use-dai/pull/2/commits/b409204f5c13b1e9bea340b9d91fd0c471b418c9

File: README.md (hunk 2 above)
Comment (kolya182) [reply README.md]: Fixed in https://github.com/apbendi/use-dai/pull/2/commits/55bf547833a3795f45d711ec79fd8d62fdddbbeb

File: README.md (hunk 3 above)
Comment (kolya182) [reply README.md:86]: Fixed in https://github.com/apbendi/use-dai/pull/2/commits/55bf547833a3795f45d711ec79fd8d62fdddbbeb

**Pull request merged**